test/e2e/network/netpol/kubemanager.go

```diff
@@ -258,45 +268,3 @@ func (k *kubeManager) deleteNamespaces(namespaces []string) error {
 	}
 	return nil
 }
-
-// waitForHTTPServers waits for all webservers to be up, on all protocols sent in the input,  and then validates them using the same probe logic as the rest of the suite.
-func (k *kubeManager) waitForHTTPServers(model *Model) error {
```

Comment:
moving this out of here so that the things depending can live outside and the manager can implement a `i can probe` interface